src/Foundation/InstalledApp.php

```diff
@@ -115,12 +111,15 @@ private function subPath($pathName): string
      */
     public function getConsoleCommands()
     {
-        return [
-            $this->container->make(GenerateMigrationCommand::class),
-            $this->container->make(InfoCommand::class, ['config' => $this->config]),
-            $this->container->make(MigrateCommand::class),
-            $this->container->make(ResetCommand::class),
-            $this->container->make(CacheClearCommand::class),
-        ];
+        $commands = [];
+
+        // The info command is a special case, as it requires a config parameter that's only available here.
+        $commands[] = $this->container->make(InfoCommand::class, ['config' => $this->config]);
+
+        foreach ($this->container->make('flarum.console.commands') as $command) {
+            $commands[] = $this->container->make($command);
+        }
```

Comment:
This needs to replicate the following piece of logic from the deprecated event:

https://github.com/flarum/core/blob/f13f17d7c92647937745e711dd1645c0de4575dc/src/Console/Event/Configuring.php#L52-L54

Without this, I believe some of @luceos' extensions will stop working once he adopts the extender. :scream:

Reply:
Fixed, good catch!

Reply:
It should be `setLaravel($this->container)` if I am not mistaken.